### ISSUE TITLE
Fishing Event in STV not starting 

### ISSUE THREAD

**Issue by cyanpill** [Progressive335/Progressive335#345]
https://wotlkdb.com/?event=15

The fishing promoter spawns in Org, but in STV no tastyfish pools, the npcs Riggle Bassbait https://wotlkdb.com/?npc=15077 and Jang to turn them in https://wotlkdb.com/?npc=15078 should be there before the event starts, and should stay after for a few hours so you can keep turning them in. I checked several times before and after 2pm server time, didn't see any of them.

**Comment by cyanpill** [comment]
Seems to be working today. Not sure if something was changed, or maybe last time it didn't work due to crashes?

**Comment by cyanpill** [comment]
I did some further testing on 11/17. I checked at 2pm server time, the npcs had not spawned, and the pools were not spawning. Hung around, no message in chat, but at 3pm server the pools started spawning so I gathered the 40 tastyfish, found that the npcs had spawned. The npcs dialog indicated that the event was already won by someone else, which is impossible because I was the only one in the zone, I checked every few minutes. However Riggle did let me complete the quest and I have the achievement to prove it.
